.gitignore

```diff
@@ -11,6 +11,11 @@ ansible.log
 tests/logs
 tests/_cache
 tests/metadata
+ansible/plugins/*/*.pyc
 
 # Dev tools
 .vscode/
+.idea/
+
+# Credential files
+ansible/group_vars/all/secrets.json
```

Comment:
it looks like you've added those internal `.gitignore` paths to your commit, can you remove them?

Reply:
leaved necessary one and removed those brought from internal branch